nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ListNodesCommand.java

```diff
@@ -95,8 +98,12 @@ public void execute(AdminCommandContext context) {
 
         String nodeList = lnh.getNodeList();
 
-         report.setMessage(nodeList);
-        
+        report.setMessage(nodeList);
+
+        Properties extraProperties = new Properties();
```

Comment:
I added these before realising I would have to use the dotted names from the `get` command for the CLI command (`create-local-instance`). It's still a valid method however of determining the names in use, and it doesn't really hurt to have these in - I'm sure we can use these elsewhere in future.

Happy to remove though and go with dotted names for both commands